collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1236,7 +1238,7 @@ public void onClick(View v) {
                 }
 
                 if (mBackButton.isShown() && mNextButton.isShown()) {
-                    mBackButton.setEnabled(true);
+                    mBackButton.setEnabled(!firstQuestion);
```

Comment:
@paraschadha2052 
sorry to keep you waiting but I was busy... I know it's tricky scenario but I still believe we can do that easier (in one place). Try something like this in this here:
 
> if (mBackButton.isShown() && mNextButton.isShown()) {
                    try {
                        boolean firstQuestion = formController.stepToPreviousScreenEvent() == FormEntryController.EVENT_BEGINNING_OF_FORM;
                        mBackButton.setEnabled(!firstQuestion);
                        formController.stepToNextScreenEvent();
                        if (formController.getEvent() == FormEntryController.EVENT_PROMPT_NEW_REPEAT) {
                            mBackButton.setEnabled(true);
                            formController.stepToNextScreenEvent();
                        }
                    } catch (JavaRosaException e) {
                        e.printStackTrace();
                    }
                    mNextButton.setEnabled(true);
                }